### DISCUSSION TITLE
Long Range Inclusion

### DISCUSSION THREAD

**Discussion by rgb2017** [zwave-js/zwave-js-ui#3713] [General]
zwave-js-ui: 9.12.0
zwave-js: 12.5.6
controller Zooz ZST39 LR

I'm trying to include new Zooz 800 series LR devices using Smart Start and selecting Z-Wave Long Range as the protocol. None of the devices that I've tried will include as a Long Range device. If I set them to standard Z-Wave then they do include successfully. 

The log says: NWI Home ID not found in provisioning list, ignoring request...  

Is this an issue where Z-Wave JS doesn't yet have a profile for these devices and that's why LR inclusion is not working? Why does standard inclusion work if that's the case?

Devices:
ZSE41 800LR
ZSE42 800LR
ZSE44 800LR
ZEN14 800LR
ZEN32 800LR

**Comment by kpine** [comment]
Did you set the controller region to US (Long Range)? Did you create the two required LR security keys?

**Reply by rgb2017** [reply]
RF Region is USA and all of the keys, including the two LR keys were set right after adding the controller.

**Reply by kpine** [reply]
Using LR requires the Long Range region.

**Reply by kpine** [reply]
To clarify, the region is literally called "USA (Long Range)". The "USA" region does not support long range.

**Reply by rgb2017** [reply]
That did the trick. That specific region was further down in the list and I didn't know it existed.  Appreciate the help!

**Reply by kpine** [reply]
I  believe the regions are sorted by their raw integer values. It would probably make more sense for ZUI to sort them alphabetically.

**Reply by robertsLando** [reply]
https://github.com/zwave-js/zwave-js-ui/issues/3714
